app/models/additional_code.rb

```diff
@@ -39,4 +46,23 @@ def code
   def id
     additional_code_sid
   end
+
+  def applicable?
+    type != UNKNOWN_TYPE
+  end
+
+  def type
+    return PREFERENCE_TYPE if additional_code_type_id.in?(PREFERENCE_TYPE_IDS)
+    return REMEDY_TYPE if  additional_code_type_id.in?(REMEDY_TYPE_IDS)
+
+    UNKNOWN_TYPE
+  end
+
+  def self.additional_codes
+    @additional_codes ||=
+      begin
+        file = File.join(::Rails.root, 'db', 'additional_codes.json').freeze
```

Comment:
Can't we use an API endpoint here instead? Seems a bit worrying that we have to resort to reading files for this.

Reply:
Yeah, this is another way of saying what we were talking about in that Slack Thread. For now, we're the source of truth for overriding annotations, types and headings. This is the same for the annotations around measure units, sadly

Reply:
Yep as we discussed let's keep this for now due to time constraints. But as a second iteration let's add this section being managed from our Admin Application.